userspace/libsinsp/metrics_collector.h

```diff
@@ -46,39 +48,6 @@ struct sinsp_stats_v2
 	uint32_t m_n_containers;
 };
 
-enum sinsp_stats_v2_resource_utilization
-{
-	SINSP_RESOURCE_UTILIZATION_CPU_PERC = 0, ///< Current CPU usage, `ps` util like calculation for the calling process (/proc/self), unit: percentage of one CPU.
```

Comment:
Any chance to keep the comments somewhere? Perhaps when we emplace_back the metrics? Just so the next developer understands better where they come from. A handful of the metrics benefit from the comments for sure.

Reply:
Agree with Melissa!

Reply:
Sounds good, I'll find a new home for them!

Reply:
I added the comments back, but instead of doing so where we emplace the variables, I added them next to the variables that store the values. I feel this closer resembles the original positions for the comments and has the benefit that it adds doxygen documentation for those fields.
I also used [member groups](https://www.doxygen.nl/manual/grouping.html#memgroup) to prevent duplicating some comments, this one is a bit nastier and I'm not fully convinced it's the right way to go about it, let me know what you think.